docs/make.jl

```diff
@@ -1,5 +1,10 @@
 using Documenter, CuArrays
 
+using Pkg
+if haskey(ENV, "GITLAB_CI")
+    Pkg.develop.(["CUDAapi", "GPUArrays", "CUDAnative", "NNlib", "CUDAdrv"])
```

Comment:
I would add these at the master branch instead (and also fuse it to a single call to `Pkg.add`)

```jl
Pkg.add([PackageSpec(name = x; rev = "master") for x in ["CUDAapi", "GPUArrays", "CUDAnative", "NNlib", "CUDAdrv"]])```

Reply:
Ah yeah that's probably better, I had changed it to `develop` in an attempt to test the issue (where I did the `Pkg.add` as part of `Pkg.build`).